src/runtime.rs

```diff
@@ -396,6 +396,15 @@ where
             .report_untracked_read(self.current_revision());
     }
 
+    /// Fores the durability of the current query to at most `durability`.
```

Comment:
```suggestion
    /// Acts as though the current query had read an input with the given durability; this will force the current query's durability to be at most `durability`.
```